test/Interop/Cxx/static/Inputs/module.modulemap

```diff
@@ -0,0 +1,15 @@
+module GlobalVar {
+  header "global-var.h"
+}
```

Comment:
I think `global-var.h` is in a different directory now.

Reply:
Cleaned up.